mslib/mscolab/models.py

```diff
@@ -65,13 +65,13 @@ class User(db.Model):
     authentication_backend = db.Column(db.String(255), nullable=False, default='local')
 
     def __init__(self, emailid, username, password, confirmed=False, confirmed_on=None, authentication_backend='local'):
-        self.username = username
-        self.emailid = emailid
+        self.username = str(username)
```

Comment:
Why are these casts necessary?

Reply:
wanted to force the type defined in the database, but str is harmless.  In the int case it woud crash when it can't cast. I am not sure if it crashes later.